agent/setup.go

```diff
@@ -105,7 +105,7 @@ func NewBaseDeps(configLoader ConfigLoader, logOut io.Writer) (BaseDeps, error)
 	d.ConnPool = newConnPool(cfg, d.Logger, d.TLSConfigurator)
 
 	builder := resolver.NewServerResolverBuilder(resolver.Config{})
-	registerWithGRPC(builder)
```

Comment:
Is this function now dead code outside of tests? Can it be deleted/moved now?

Reply:
Ah yes! I thought I had removed it, but looks like I missed it. Good catch!

Reply:
Done